src/main/java/com/github/tomakehurst/wiremock/common/ClasspathFileSource.java

```diff
@@ -52,11 +52,22 @@ public ClasspathFileSource(String path) {
         this.path = path;
 
         try {
-            URL resource = firstNonNull(
+            ClassLoader cl = firstNonNull(
                     currentThread().getContextClassLoader(),
-                    Resources.class.getClassLoader())
-                        .getResource(path);
+                    Resources.class.getClassLoader()
+            );
+            URL resource = cl.getResource(path);
+
+            // If the resource didn't exist, and the path included a leading slash, try it without.
+            // (When WireMock is packaged inside a runnable jar, frequently the 'mappings' directory may be at the root
+            // level. Some users then attempt 'usingFilesUnderClasspath("")' or 'new ClasspathFileSource(""). Since the
+            // #child method uses 'path + "/" + subDirectoryName', the child ClasspathFileSource then uses "/mappings".
+            // That leading slash will not work in this scenario.)
+            if (resource == null && path.startsWith(File.separator)) {
+                resource = cl.getResource(path.replaceFirst(File.separator, ""));
```

Comment:
Review note: not removing the leading slash from `this.path` at this point -- unsure of side-effects.